pinot-perf/src/main/java/org/apache/pinot/perf/BenchmarkRangeIndex.java

```diff
@@ -106,7 +105,7 @@ public void setup()
           break;
         }
         case FLOAT: {
-          DoubleSupplier supplier = Distribution.createDoubleSupplier(_seed, _scenario);
+          Distribution.DataSupplier supplier = Distribution.createSupplier(_seed, _scenario);
```

Comment:
nit: why are you using `Distribution.DataSupplier` in some places and `LongSupplier` in other places?

Reply:
I didn't want to change all of the old code that used LongSupplier (which DataSupplier does implement).